### ISSUE TITLE
Css style use display: none hides use tags on entire page

### ISSUE THREAD

**Issue by albertogasparin** [kisenka/svg-baker#2]
By default you [add a style tag](https://github.com/kisenka/svg-baker/blob/18b2340c0da957204bd5ba4338cc01939048c8b4/packages/svg-baker/lib/sprite-factory.js#L12
) on the generated svg to hide the `use` tags that are not the current tag.
The problem with that selector is that if we use an universal svg sprite loading technique (so inlining the svg contents on the page) that stile actually hides all the `use` tags on the site.

The code I'm using to load the svg in the page is this one (but in another project I'm using the server to print the svg directly on the page):
``` html
<object type="image/svg+xml" data="/assets/sprite.svg" onload="this.parentNode.replaceChild(this.getSVGDocument().childNodes[0], this); />"
```

A solution could be adding a group and hiding just the `use` tags inside that group, so it will not affect any other svg on the page.
```svg
<defs>
  <style>.svg-baker-sprite > use {display: none;} .svg-baker-sprite > use:target {display: inline;}</style>
  <symbol viewBox="0 0 24 24" id="i-add"><path d="M17 13h-4v4h-2v-4H7v-2h4V7h2v4h4m2-8H5c-1.11 0-2 .89-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2V5a2 2 0 0 0-2-2z" /></symbol>
</defs>
<g class="svg-baker-sprite">
  <use id="i-add-usage" xlink:href="#i-add" />
</g>
```

**Comment by kisenka** [comment]
Oh shi~! Thanks for reporting @albertogasparin, will fix it asap!

**Comment by kisenka** [comment]
Shipped with `svg-baker@1.2.7`
